docs/io/fits/appendix/faq.rst

```diff
@@ -329,17 +313,26 @@ it, because it doesn't happen until the data portion of the HDU is accessed
 (to allow things like updating the header without rescaling the data).  For
 example::
 
-    >>> hdul = fits.open('scaled.fits')
-    >>> image = hdul['SCI', 1]
+    >>> from astropy.utils.data import get_pkg_data_filename
+    >>> def _(filename):
```

Comment:
This is clever but I think for many users it may end up distracting from the example. Personally I don't think there's anything wrong with just having something like ```filename = get_pkg_data_filename(...)``` and then passing `filename` to `fits.open`, which you did in many of the other examples. A brief code comment above the call to `get_pkg_data_filename` explaining why this is different from the real usage would be fine.

Reply:
oh, I missed that one? That was my first attempt. I'll revert it.

Reply:
It would be good to have a more concise way to get the data filenames.
Maybe you could add a `io.fits` specific utility function, as it is used everywhere in the docs. Something like:
```
def get_fits_data_filename(filename):
    return get_pkg_data_filename('io/fits/tests/data/{}'.format(filename), 'astropy')
```
Many packages have similar utility functions to get the test data files.

Reply:
@saimn I thought about that too. But then I already converted 80% of the docs and wanted to wait for comments before I change the overall approach.

Reply:
IMO it would be much cleaner to use something like `fits.open(get_fits_data_filename('test0.fits'))`. @drdavella, what's your opinion ?

Reply:
I'll make a commit that introduces that utility later. 😄

Reply:
circleci is linux, too ;)

Reply:
Oups, I meant Travis and CircleCI 😄